state-synchronizer/src/synchronizer.rs

```diff
@@ -60,7 +61,7 @@ impl StateSynchronizer {
             network,
             state_sync_to_mempool_sender,
             config.base.role,
-            config.base.waypoint,
+            Some(waypoint),
```

Comment:
thanks for fixing it, why not fix all the way?

Reply:
not the right PR for that :D